examples/create-react-app/WORKSPACE

```diff
@@ -20,7 +20,13 @@ http_archive(
     ],
 )
 
-load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+
+# M1 Macs require Node 16+
+node_repositories(
```

Comment:
the issue is present in all examples and applications using rules_nodejs, so let's not just fix it in one place.
I think https://github.com/bazelbuild/rules_nodejs/pull/3107 is more principled

Reply:
Sounds good!